app/Router.tsx

```diff
@@ -65,7 +67,13 @@ export const Router = ({
       <Route
         exact
         path="/organizations/:id"
-        component={OrganizationListingPage}
+        component={() => (
+          <SecondaryNavigationLayout
```

Comment:
I'm liking this approach! Seems like an easy way to reuse this component for events/opportunities/misc pages